code/controllers/subsystem/pai.dm

```diff
@@ -24,6 +24,8 @@ SUBSYSTEM_DEF(pai)
 				pai.name = candidate.name
 			pai.real_name = pai.name
 			pai.key = candidate.key
+			pai.greyscale_colors = "[candidate.color][candidate.secondary_color]"
```

Comment:
This might make it so we don't have to save that secondary color.
```suggestion
			pai.greyscale_colors = "[candidate.color][invert_hex(candidate.color)]"
```